### PR TITLE
Potential fix for packaging

### PR DESCRIPTION
**Pull request by daineAMD**
Into release/21.40 this time.

### CODE REVIEW

**Review by TorreZuk — APPROVED** [review]
LGTM.  So now will you close the original and after merge, then do a PR to merge release/21.40 branch back into develop much like we do with linux master?

**Comment by daineAMD** [comment]
Yeah, that was the plan!

**Comment by cgmb** [comment]
Just for future reference: if you click the `edit` button directly to the right of the Pull Request title, you can change the branch that a PR is targeting without having to create a new one.

File: rmake.py
Comment (cgmb) [inline rmake.py:104]: It's purely a matter of style, I would suggest using `'` quotes when the string contains `"` and use `"` when the string contains a `'` to minimize the need for backslash escapes. See [PEP 8](https://www.python.org/dev/peps/pep-0008/#string-quotes).

This file is probably still undergoing rapid changes across in multiple repositories, so it's probably a waste of time to make those sorts of tiny improvements. But, I figured I should mention it anyway, for when the time comes.

**Review by cgmb — COMMENTED** [review]
Nobody's tested this, I take it? I suppose I'll give it a try.

**Comment by daineAMD** [comment]
I see the nrm2 failures that were fixed in https://github.com/ROCmSoftwarePlatform/hipBLAS/pull/370 are failing. Can I cherry-pick that commit into this branch as well?

**Review by cgmb — APPROVED** [review]
I'm having difficulty reproducing the problem, so I'll just defer to the Windows CI. That's where the problem was anyway. LGTM, if you can get a passing build on the Windows CI.

**Comment by saadrahim** [comment]
> 
> 
> I see the nrm2 failures that were fixed in #370 are failing. Can I cherry-pick that commit into this branch as well?

Yes, please cherry-pick that commit as well.

**Comment by TorreZuk** [comment]
@saadrahim are you planning on moving all the 21.40 branches for all repo again, otherwise I think cherry-picking fixes is fine by me.   What date were these branches all based off , or off the current master for 4.5 ?  @amcamd guess you should be in the loop, as need any branch for release managed.  thanks

**Comment by saadrahim** [comment]
> 
> 
> @saadrahim are you planning on moving all the 21.40 branches for all repo again, otherwise I think cherry-picking fixes is fine by me. What date were these branches all based off , or off the current master for 4.5 ? @amcamd guess you should be in the loop, as need any branch for release managed. thanks

Not sure what you mean by moving @TorreZuk . Fixes are going is fast as we can put them in. Other than that, I don't really want to advance the 21.40 branch any further. 21.40 was branched Friday based on master at that time. Around 3-4 pm MST.

**Comment by TorreZuk** [comment]
Okay if the plan is just manual cherry-picks that is still a plan, just saying blas, solver, hipBLAS normally must be kept in sync so if this was all master~=4.5 based that was the original synchronization that should be fine.  just need to keep 21.40 from diverging.

**Comment by TorreZuk** [comment]
Or should have said diverging "too much"  :-)   Otherwise merging master to 21.40 and 21.40 back into master we are saying is not planned.  21.40 is a dead end branch from what I am hearing, it is taking cherry picks from "5.0"~develop so can't be merged back to master.  This should be documented in a wiki or channel.

**Pull request merged**
